### PR TITLE
Churchship

### PR DESCRIPTION
**Pull request by AsciiSquid**
## About The Pull Request
chapel

## Why It's Good For The Game
No swearing on the Christian minecraft ship

## Changelog
:cl:
add: Added a new ship, the Shepherd-Class
/:cl:

### CODE REVIEW

**Comment by Apogee-dev** [comment]
Excellent ship and I'm greatly looking forward to going to space church. However, it's spelled "Shepherd."

**Comment by AsciiSquid** [comment]
> However, it's spelled "Shepherd."

image

**Comment by Pontenerd** [comment]
People got sick of bringing hozer to church so they made this to bring the church to him

**Comment by HighPrincessErinys** [comment]
i really like those diagonal wires

File: _maps/configs/chapel.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+	"map_name": "Shepherd-Class Space Monastery",
+	"map_short_name": "Space Church",
```
Comment (triplezeta) [inline _maps/configs/chapel.json]: please make the short name the map's ship class (in this case `Shepherd-class`) thanks :)

Comment (AsciiSquid) [reply _maps/configs/chapel.json]: please make single-line edit requests a code suggestion
```suggestion
	"map_short_name": "Shepherd-class",
```
thanks :)

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Comment by shiptest-server** [comment]
#### Test Merge Deployed

##### Server Instance
shiptest

##### Revision
Origin: aabc6f1b144d0c36cf9f9f5d49313c6586ec6706
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: 4a612b6abc77d3aaae270b65dc4c94667e4d6be2

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 293bad3e54aa3f26644d5d871baee9eda36057ca
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: 2c17e761a3dbe846412b0eb5224178d8c7dfc1e4

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: d79659920aec58007df63d3e9b96cb0fe1a16353
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: 72d94bef2620899bd315f660ccb547df14ffc328

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: dd854e7386a7a9a664d6357aab6213b032e1817b
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: d097fa6e32a13458e756307a43c1764f99b243bc

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: 4d98cdcea7b4591b94a06237309039f4980217fa
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: 9a29cfa82da92d64f18a8f8bcd69d3ea934be1ac

**Comment by shiptest-server** [comment]
#### Test Merge Updated

##### Server Instance
shiptest

##### Revision
Origin: f789e74004b3acb82690d77398848b240a153b6f
Pull Request: ffc4427c9d2c95c6b486a1ba96f937d101d25312
Server: fec71ade6e716fb63d267abca95b86e22f5eb182

**Comment by shiptest-server** [comment]
#### Test Merge Removed

**Pull request merged**